Split/Telemetry/Storage/TelemetryConfig.swift

```diff
@@ -11,13 +11,15 @@ import Foundation
 struct TelemetryRates: Codable {
     var splits: Int
     var mySegments: Int
+    var myLargeSegments: Int
     var impressions: Int
     var events: Int
     var telemetry: Int
 
     enum CodingKeys: String, CodingKey {
         case splits = "sp"
         case mySegments = "se"
+        case myLargeSegments = "ls"
```

Comment:
it is `mls` actually. We made a typo in the spec.

Reply:
👍 Thank you doc!